cityofaustin/janis
# 3282 manual incremental builds

# Description

Adds some logic to our builds to allow for triggering an incremental build

[Joplin Related Pull Request Link](https://github.com/cityofaustin/joplin/pull/468)

# Types of changes



# How can this be tested?
1. Run joplin from https://github.com/cityofaustin/joplin/pull/468
2. Build the site using `./scripts/build-full.sh`
3. Update a page in local joplin
4. Get the page ID from the Joplin terminal
5. Set `PAGE_ID` in `build-incremental.sh`
6. Build the site incrementally using `./scripts/build-incremental.sh`

After doing this you should see your changes on the local instance of the site.

# Checklist:
